src/TrackRowZoomOverlay.js

```diff
@@ -0,0 +1,65 @@
+import React, { useRef, useEffect } from 'react';
+import debounce from 'lodash/debounce';
+import './TrackRowZoomOverlay.scss';
+
+/**
+ * An overlay element to intercept wheel events in horizontal multivec tracks.
+ * @prop {number} trackX The x offset of the horizontal multivec track.
+ * @prop {number} trackY The y offset of the horizontal multivec track.
+ * @prop {number} trackWidth The width of the horizontal multivec track.
+ * @prop {number} trackHeight The height of the horizontal multivec track.
+ * @prop {function} onZoomRows The function to call on zoom events.
+ * @prop {boolean} isWheelListening Should the wheel events be intercepted right now?
+ * i.e. is the designated key pressed down?
+ */
+export default function TrackRowZoomOverlay(props) {
```

Comment:
Can you add this function to the documentation?

Reply:
Good catch, thanks!